launchUI.py

```diff
@@ -1568,69 +1565,72 @@ def place_xbar_options(master):
 
 def create_param_pannel(master):
     paramPannel = tkinter.LabelFrame(master, text='Configuration', bd=BORDER, relief='groove')
-    paramPannel.grid(row=0,column=1,rowspan=1,columnspan=1,sticky="nsew")
+    paramPannel.grid(row=0, column=1, rowspan=1, columnspan=1, sticky="nsew")
     for i in range(10):
-        if i < 8:
-            paramPannel.rowconfigure(i, weight=1)
-    else:
-            paramPannel.rowconfigure(i, weight=2)
-    for i in range(4):
-        paramPannel.columnconfigure(i,weight=1)
+        paramPannel.rowconfigure(i, weight=1)

```

Comment:
Please comment on `weight`.

Reply:
Okay. Here the entire paramPannel is divided into many rows and columns, and the weights are allocated according to the size of the components to be occupied. The “weight” represents the weight of the corresponding row/column length.

Reply:
Ah, i mean add the comment above the corresponding code.

Reply:
Okay I will do it.

Reply:
There were some issues with my local environment, and a clash occurred, leading to an inability to sync the changes. I will submit the modified version in the future.